features/clipboard-events.yml

```diff
@@ -11,8 +13,6 @@ compat_features:
   - api.Element.copy_event
   - api.Element.cut_event
   - api.Element.paste_event
-  - api.Permissions.permission_clipboard-write
-
 # The following features in the spec are already part of web-features:
 # - Async clipboard:
 #   - api.Clipboard
```

Comment:
This comment block needs to be removed. These are automatically generated only for the draft spec features, so they won't be updated as regular features. They're quickly go stale, so let's drop them whenever converting a draft.

Reply:
Got it, removed!